src/compiler/compile-xslt-tests.xsl

```diff
@@ -191,13 +192,13 @@
          </xsl:for-each>
       </call-template>
 
-      <!-- Continue compiling calls. -->
+      <!-- Continue invoking the compiled x:scenario or x:expect. -->
```

Comment:
The word "the" and lack of plural makes it sound a bit as if you're continuing to invoke the same compiled `x:scenario` or `x:expect` that brought you to this template in the first place. How about `Continue invoking compiled x:scenario or x:expect elements`?

This same comment appears in `compile-xquery-tests.xsl`, too.

Reply:
Thank you for reviewing this, @galtm.
Fixed in f60ad9c630cf868e2261f09ddc0199d75aad7d4b.